assets/jss/app.js

```diff
@@ -0,0 +1,104 @@
+/*const contactNav = document.getElementById("contactNav");
+const aboutNav = document.getElementById("aboutNav");
+const aboutMe = document.getElementsByClassName("p-text")[0];

```

Comment:
do `document.getQuerySelector('p-text')` which only selects the first by default.